ods_ci/tests/Resources/Files/pipeline-samples/ray_integration.py

```diff
@@ -0,0 +1,86 @@
+from kfp import components, dsl
+from ods_ci.libs.DataSciencePipelinesKfpTekton import DataSciencePipelinesKfpTekton
+
+
+def ray_fn(openshift_server:str, openshift_token:str) -> int:
+    from codeflare_sdk.cluster.cluster import Cluster, ClusterConfiguration
+    from codeflare_sdk.cluster.auth import TokenAuthentication
+    import ray
+
+
+    auth = TokenAuthentication(
+        token=openshift_token,
+        server=openshift_server,
+        skip_tls=True
+    )
+    auth.login()
+    cluster = Cluster(ClusterConfiguration(
+        name='raytest',
+        namespace='dlovison-ray',
+        num_workers=1,
```

Comment:
```suggestion
        head_cpus='500m',
        head_memory=1,
        num_workers=1,
```
This limits resources for Ray head pod.

Reply:
Done. Fix also the namespace